youtube_dl/extractor/extractors.py

```diff
@@ -98,6 +98,7 @@
 from .baidu import BaiduVideoIE
 from .bandaichannel import BandaiChannelIE
 from .bandcamp import BandcampIE, BandcampAlbumIE, BandcampWeeklyIE
+from .bandlab import BandlabIE, BandlabAlbumOrPlaylistIE
```

Comment:
Easier to maintain (actually, Bandcamp, above, is one of the few remaining one-liners):
```suggestion
from .bandlab import (
    BandlabAlbumOrPlaylistIE,
    BandlabIE,
)
```